src/BAP/SimpleBTSBundle/Migrations/Data/Demo/ORM/LoadIssueData.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace BAP\SimpleBTSBundle\Migrations\Data\Demo\ORM;
+
+use BAP\SimpleBTSBundle\Entity\Issue;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\DataFixtures\AbstractFixture;
+use Doctrine\Common\Persistence\ObjectManager;
+use Oro\Bundle\UserBundle\Entity\User;
+
+class LoadIssueData extends AbstractFixture
+{
+    public function load(ObjectManager $manager)
+    {
+        /** @var ArrayCollection|User[] $users */
+        $users = $manager->getRepository('OroUserBundle:User')->findBy([], ['id' => 'asc'], 1);
```

Comment:
Please add some additional users and set issues to random users. In this case, your data will be much interesting than now.